examples/spark/pyspark_udfs/pandas_udfs.py

```diff
@@ -0,0 +1,52 @@
+"""Pandas UDFs.
+
+Has to only contain map operations! Aggregations, filters, etc. are not supported.
```

Comment:
What will happen if its not just map operations?

Reply:
pandas udfs only have certain shapes -- so it'll break in pyspark.